exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorker.java

```diff
@@ -93,27 +104,44 @@ boolean registerView(View view) {
     }
     registeredViews.put(view.getName(), view);
 
-    // TODO(songya): don't need to create MetricDescriptor for RpcViewConstants once we defined
-    // canonical metrics. Registration is required only for custom view definitions. Canonical
-    // views should be pre-registered.
-    MetricDescriptor metricDescriptor =
-        StackdriverExportUtils.createMetricDescriptor(view, projectId);
-    if (metricDescriptor == null) {
-      // Don't register interval views in this version.
-      return false;
-    }
+    Span span = tracer.getCurrentSpan();
```

Comment:
You need to call span.end() at one point.

Reply:
Right I forgot this - I added it at the end of export().